clients/cli/src/edit.rs

```diff
@@ -121,3 +129,57 @@ pub fn edit(file_name: &str) {
         )
     });
 }
+
+pub fn save_file_to_core(
```

Comment:
Let's move this to `utils.rs`. Consider renaming it to `save_temp_file_contents`.